jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/Response.java

```diff
@@ -317,9 +341,22 @@ static void sendRedirect(Request request, Response response, Callback callback,
             }
         }
 
-        response.getHeaders().put(HttpHeader.LOCATION, toRedirectURI(request, location));
+        if (content == null && request.getConnectionMetaData().getHttpConfiguration().isGenerateRedirectBody())
+        {
+            response.getHeaders().put(MimeTypes.Type.TEXT_HTML_8859_1.getContentTypeField());
+            String body = """
+            <!DOCTYPE html>
+            <html lang="en">
+            <head><meta charset="ISO-8859-1"/><meta http-equiv="refresh" content="0; URL=%s/"/><title>Redirecting...</title></head>
```

Comment:
There is an extra "/":

```suggestion
            <head><meta charset="ISO-8859-1"/><meta http-equiv="refresh" content="0; URL=%s"/><title>Redirecting...</title></head>
```